mqttclients/c/mosquitto_client_extensions/json_handlers/geo_json_handler.c

```diff
@@ -9,42 +9,42 @@
 
 #include "geo_json_handler.h"
 
-#define RETURN_IF_NULL(x, jobj_to_free)               \
-  do                                                  \
-  {                                                   \
-    if ((x) == NULL)                                  \
-    {                                                 \
-      printf("JSON Parsing Error: %s is NULL\n", #x); \
-      if (jobj_to_free != NULL)                       \
-      {                                               \
-        json_object_put(jobj_to_free);                \
-      }                                               \
-      return -1;                                      \
-    }                                                 \
+#define RETURN_IF_NULL(x, jobj_to_free)                         \
+  do                                                            \
+  {                                                             \
+    if ((x) == NULL)                                            \
+    {                                                           \
+      printf("[ERROR] Failure parsing JSON: %s is NULL\n", #x); \
```

Comment:
See my other PR recommendation to instead make this a macro 
`printf( ERROR "error text");`

or even better a different signature:

`log_error / log_warn / log_info / log_verb` with compile-time switches to enable/disable some of the logging.

Reply:
Done!